observability/deployment/fluent-bit.yaml

```diff
@@ -143,6 +200,22 @@ config:
             tls.verify Off
             Trace_Error On
 
+        [OUTPUT]
+            Name opensearch
+            Host opensearch-data
+            HTTP_Passwd admin
+            HTTP_User admin
+            Logstash_Format On
+            Logstash_DateFormat %Y-%m-%d
+            Logstash_Prefix wso2-apim-application-logs
+            Match_regex ^wso2_apim_carbon_logs$
```

Comment:
No need to use Match_regex here. Just a Match with the exact tag name is enough.

Reply:
Done via https://github.com/wso2/observability-resources/pull/33/commits/ce8f39c772df9ff67a425abbeb810cfa07c9b029